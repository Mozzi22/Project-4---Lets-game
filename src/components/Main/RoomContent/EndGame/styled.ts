import styled from 'styled-components';

export const StEndGame = styled.div`
        display:flex;
        justify-content: center;
        align-items: center;
        width: 100%;
        height: 300px;
        color: ${({ colors, theme }) => colors[theme].focusColorDefault};

`;