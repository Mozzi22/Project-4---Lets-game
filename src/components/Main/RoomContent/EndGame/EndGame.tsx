import React from 'react';
import { StEndGame } from './styled';
import { useTheme } from 'src/components/Hooks/useTheme';

function EndGame({ winner }) {
const { colors, theme } = useTheme();
    return (
        <StEndGame theme={theme} colors={colors}>
            {winner}
        </StEndGame>
    );
}

export default EndGame;
