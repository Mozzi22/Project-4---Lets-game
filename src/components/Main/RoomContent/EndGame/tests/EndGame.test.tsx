import React from 'react';
import { shallow, mount } from 'enzyme';
import EndGame from '../EndGame';

describe('EndGame', () => {
    let props;
    beforeEach(() => {
        props = {
            winner: 'a',
            userLogin: 'aa',
        };
    });
    it('Should match snapshot', () => {
        const component = shallow(<EndGame {...props}/>);
        expect(component.html()).toMatchSnapshot();
    });
    it('Should render StEndGame', () => {
        props = {
            winner: 'a',
            userLogin: 'a',
        };
        const component = mount(<EndGame {...props} />);
        expect(component.find('styled__StEndGame')).toHaveLength(1);
    });
});
