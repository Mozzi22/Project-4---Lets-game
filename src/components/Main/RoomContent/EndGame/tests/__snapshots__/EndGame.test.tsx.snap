// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EndGame Should match snapshot 1`] = `"<div class=\\"styled__StEndGame-sc-qa3399-0 gWauTJ\\">a</div>"`;
